server/base/src/main/java/org/apache/accumulo/server/AbstractServer.java

```diff
@@ -181,26 +180,15 @@ public AtomicBoolean getShutdownComplete() {
    * </pre>
    */
   public void runServer() throws Exception {
-    final AtomicReference<Throwable> err = new AtomicReference<>();
     serverThread = new Thread(TraceUtil.wrap(this), applicationName);
-    serverThread.setUncaughtExceptionHandler((thread, exception) -> err.set(exception));
+    serverThread.setUncaughtExceptionHandler(Threads.UEH);
```

Comment:
I think change is might have better handling for `Error` than what is used at https://github.com/apache/accumulo/blob/2ecf4f69419570a060a1ba191ab0a0a00002fef9/start/src/main/java/org/apache/accumulo/start/Main.java#L157-L170

Reply:
Did not realize all of that code in start.Main would eventually be called.  Reverted the changes to AbstractServers so that things would keep working the same way between AbstractServer and Main.start